### PR TITLE
fix(osc): osc job is terminated when database id is not exist

### PR DESCRIPTION
**Pull request by krihy**
#### What type of PR is this?

type-bug
module-schedule
#### What this PR does / why we need it:
because schedule task input param from osc do not contain project id, so quartz job will be deleted by  `ScheduleService#terminateIfDatabaseNotExisted`

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by guowl3 — DISMISSED** [review]
LGTM

**Review by LuckyPickleZZ — APPROVED** [review]
lgtm

File: server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/OnlineSchemaChangeFlowableTask.java [hunk 1]
```diff
@@ -267,9 +269,9 @@ private ScheduleEntity createScheduleEntity(ConnectionConfig connectionConfig,
         scheduleEntity.setAllowConcurrent(false);
         scheduleEntity.setCreatorId(creatorId);
         scheduleEntity.setOrganizationId(organizationId);
-        // todo project id database id
+        // todo project id
         scheduleEntity.setProjectId(1L);
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/OnlineSchemaChangeFlowableTask.java:273]: what is projectId = 1 means ?

Comment (krihy) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/OnlineSchemaChangeFlowableTask.java]: project id is not used for osc , but i also query by `databaseService.getBasicSkipPermissionCheck` and and set in scheduleEntity

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by LuckyPickleZZ — APPROVED** [review]

**Pull request merged**